packages/react-core/src/components/ToggleGroup/examples/ToggleGroupIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import UndoIcon from '@patternfly/react-icons/dist/esm/icons/undo-icon';
+import CopyIcon from '@patternfly/react-icons/dist/esm/icons/copy-icon';
+import ShareSquareIcon from '@patternfly/react-icons/dist/esm/icons/share-square-icon';
+
+export const ToggleGroupIcon: React.FunctionComponent = () => {
+  const [state, setState] = React.useState({
+    isSelected: {
+      'icons-1': false,
+      'icons-2': false,
+      'icons-3': true
+    }
+  });
+  const handleItemClick = (isSelected, event) => {
+    const id = event.currentTarget.id;
+    setState(prevState => {
+      prevState.isSelected[id] = isSelected;
+      return {
+        isSelected: prevState.isSelected
+      };
+    });
+  };
+  return (
+    <ToggleGroup aria-label="Icon variant toggle group">
+      <ToggleGroupItem
+        icon={<CopyIcon />}
+        aria-label="copy icon button"
```

Comment:
For the ToggleGroupItem's in this example, we could omit the "icon button" part of the aria-label since assistive tech should already announce them as "toggle button".